ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -290,11 +397,13 @@ def foo1(v, amount):
             return v + 1
 
 
-@pytest.mark.only_on_backends(['pandas', 'pyspark', 'dask'])
+@pytest.mark.only_on_backends(['pandas', 'pyspark'])
+@pytest.mark.xfail_backends(['dask'])
```

Comment:
Do we already have an existing issue for the dask destructure refactoring work that would be required here? @gerrymanoim